easylist_adult/adult_specific_hide.txt

```diff
@@ -37,6 +37,8 @@ euroxxx.net###block-15
 18adultgames.com###block-20
 hentaiprn.com###block-27
 jav-jp.com###block-29
+ryuugames.com###block-34
+ryuugames.com###block-39
```

Comment:
https://www.ryuugames.com/tropical-kiss-free-download/

![CleanShot 2025-02-28 at 20 03 35](https://github.com/user-attachments/assets/75d9cf63-dd6d-456d-a8fd-f851188ef235)

Reply:
Frankly, all of them except for `###wpfront-notification-bar-spacer` look insignificant to EL and for such a site. AG/uAssets may add them (`ryuugames.com###block-34` is in AG Base), but these will later become useless and only waste RAM. EL tends to be more selective in this regard.